src/lib/notificationHandlerWebsocket.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const _ = require('lodash')
+const EventEmitter = require('events').EventEmitter
+const transferDictionary = require('five-bells-shared').TransferStateDictionary
+const transferStates = transferDictionary.transferStates
+
+class NotificationHandler extends EventEmitter {
+  constructor (log, Fulfillment, config) {
+    super()
+    this.log = log
+    this.Fulfillment = Fulfillment
+    this.config = config
+  }
+
+  * sendNotifications (transfer, transaction) {
+    const affectedAccounts = _([transfer.debits, transfer.credits])
+      .flatten().pluck('account').value()
+    affectedAccounts.push('*')
+
+    // Prepare notification for websocket subscribers
+    const notificationBody = {
+      resource: transfer.getDataExternal()
+    }
+
+    // If the transfer is finalized, see if it was finalized by a fulfillment
+    let fulfillment
+    if (transfer.isFinalized()) {
+      fulfillment = yield this.Fulfillment.findByTransfer(transfer.id, { transaction })
```

Comment:
Are you working off an old copy of the ledger? I don't see `findByTransfer` anywhere in the code.